src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -309,6 +309,17 @@ function WorkspaceCategoriesPage({route}: WorkspaceCategoriesPageProps) {
                     Navigation.navigate(ROUTES.WORKSPACE_CATEGORIES_IMPORT.getRoute(policyId));
                 },
             },
+            {
+                icon: Expensicons.Download,
+                text: translate('common.downloadCSV'),
```

Comment:
add the translation key to `spreadsheet` instead of `common`

Reply:
I was following "Import spreadsheet" example as `common.importSpreadsheet`; should I move it to spreadsheet too?

Reply:
Yes, do it please 👌